packages/SwingSet/src/controller/controller.js

```diff
@@ -215,11 +215,18 @@ export async function makeSwingsetController(
   function kernelRequire(what) {
     Fail`kernelRequire unprepared to satisfy require(${what})`;
   }
+  const kernelConsole = makeConsole(`${debugPrefix}SwingSet:kernel`);
+  const sloggingKernelConsole = makeLimitedConsole(level => {
+    return (...args) => {
+      kernelConsole[level](...args);
+      writeSlogObject({ type: 'console', source: 'kernel', args });
```

Comment:
This is missing the level

```suggestion
      writeSlogObject({ type: 'console', source: 'kernel', level, args });
```

Reply:
Nice catch!
#11052